tensorflow/core/kernels/data/name_utils.cc

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#include "tensorflow/core/kernels/data/name_utils.h"
+
+namespace tensorflow {
+namespace data {
+namespace name_utils {
+
+string ToString(const DatasetType& dataset_type) {
+  switch (dataset_type) {
+    case RANGE_DATASET:
```

Comment:
no need for the `_DATASET` suffix, it is implied by the enum type

Reply:
Done